src/test/java/org/minions/devfund/bruno/WhackAMoleTest.java

```diff
@@ -0,0 +1,236 @@
+package org.minions.devfund.bruno;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * This class in charged of testing the whack a mole game.
+ */
+public class WhackAMoleTest {
+    private static final int NUM_ATTEMPTS = 50;
+    private static final int GRID_DIMENSION = 10;
+    private static final int NUM_MOLES = 10;
+    private WhackAMole game;
+
+    /**
+     * Before test.
+     */
+    @Before
+    public void beforeTest() {
+        game = new WhackAMole(NUM_ATTEMPTS, GRID_DIMENSION);
+    }
+
+    /**
+     * Verifies if the method generate a grind to print.
+     */
+    @Test
+    public void place() {
+        final int positionX = 0;
+        final int positionY = 0;
+        assertTrue(game.place(positionX, positionY));
+    }
+
+    /**
+     * Verifies if the all moles was placed.
+     */
+    @Test
+    public void placeAllMoles() {
```

Comment:
Since your pull request is still open.
please write your unit tests something like this.

Running: testConstructorGrid
Finished: testConstructorGrid
Running: testConstructorNumOfAttempts
Finished: testConstructorNumOfAttempts
Running: testPlaceFail
Finished: testPlaceFail
Running: testWhackSucceedAttempts
Finished: testWhackSucceedAttempts
Running: testWhackFailAttempts
Finished: testWhackFailAttempts
Running: testPlaceSucceed
Finished: testPlaceSucceed
Running: testWhackFailMolesLeft
Finished: testWhackFailMolesLeft
Running: testWhackSucceedMolesLeft
Finished: testWhackSucceedMolesLeft